### PR TITLE
Add help command

### PR DESCRIPTION
**Pull request by tohyuting**
Added help command as well as the respective test cases for help command.

### CODE REVIEW

File: src/main/java/seedu/clinic/logic/commands/HelpCommand.java [hunk 1]
```diff
@@ -3,19 +3,218 @@
...
+            + " respective utility description\n"
+            + "Example: " + COMMAND_WORD + "\n\nSpecific commands with respective input samples "
+            + "can be shown as well \n" + "Example: help find";
+    public static final String MESSAGE_TOO_MANY_ARGUMENTS = "You can only enter one command at a time\n\n"
+            + MESSAGE_USAGE;
+    public static final String MESSAGE_WRONG_ARGUMENT = "You can only enter command from one of the "
+            + "following keywords: add, clear, delete, exit, find, view, update";
 
     public static final String SHOWING_HELP_MESSAGE = "Opened help window.";
+    public static final String[] ALLOWED_ARGUMENT = new String[]{
```
Comment (qlchan24) [inline src/main/java/seedu/clinic/logic/commands/HelpCommand.java]: Perhaps this can be renamed to ALLOWED_ARGUMENTS or ALLOWED_ARGUMENT_LIST

**Review by qlchan24 — COMMENTED** [review]
Other than the possible naming issue, LGTM

File: src/main/java/seedu/clinic/logic/commands/HelpCommand.java (hunk 1 above)
Comment (tohyuting) [reply src/main/java/seedu/clinic/logic/commands/HelpCommand.java]: Thanks for pointing that out, will push another commit to fix this!

**Review by jeffreytjs — APPROVED** [review]
Looks good

**Pull request merged**